docs/endpoints/post-token-refresh.md

```diff
@@ -95,10 +99,10 @@ An error response might look like the following:
 
 | Property | Data Type | Description |
 | :--- | :--- | :--- |
-| `advertising_token` | string | An encrypted advertising (UID2) token for the user. |
+| `advertising_token` | string | The UID2 token (advertising token) for the user. |
```

Comment:
_The UID2 token (also known as advertising token) for the user. |_
Otherwise i feel like we were saying advertising_token is advertising token ?!

Reply:
But will leave it to you Gen :)

Reply:
I fixed this, Sunny. Used your wording basically, and added a link also.